packages/common/src/permissions/HubPermissionPolicies.ts

```diff
@@ -62,6 +62,21 @@ import { PlatformPermissionPolicies } from "./PlatformPermissionPolicies";
 //   },
 // ];
 
+const TempPermissionPolicies: IPermissionPolicy[] = [
+  {
+    permission: "hub:workspace:released",
+    subsystems: [],
+    assertions: [
+      {
+        // not prod
+        property: "context:hubUrl",
+        type: "not-ends-with",
+        value: "hub.arcgis.com",
+      },
+    ],
+  },
+];
```

Comment:
A permission policy that will only pass if context.hubUrl does not end with hub.arcgis.com. Ie it will fail in prod.

Reply:
Could we call this `flag:workspace:released` and `FlagPermissionPolicies`? or leave it as `TempPermissionPolicies` but change the string to `temp:workspace:released` ?

++ on making use of the existing assertion infrastructure :)